index.php

```diff
@@ -102,7 +103,7 @@ function mysort($a, $b)
                             }
 
                             $dl = '';
-                            if (file_exists(SHELL_ROOT . 'public/' . substr($file, 5, -5) . '.sav')) {
+                            if (file_exists(SHELL_ROOT . 'maps/' . substr($file, 5, -5) . '.sav')) {
                                 $dl = ' (DL)';
```

Comment:
this one is wrong - DL (downLoad) should only be possible for files in /saves/public.
here you are checking (maps